### PR TITLE
Move bash completion scripts for geopmread & write to service

### LINKED ISSUE #2872: The bash completion files for geopmread and geopmwrite are installed with the wrong package
**Describe the bug**
I tried to use the bash auto-completion with geopmread and geopmwrite but it was not enabled with the packages that provide the binaries (they are packaged with the geopm runtime).

**GEOPM version**
Output from `geopmread --version`
f51f7b63e99365cd6f5a8116c0534f0ba305de18

**Expected behavior**
These scripts should be installed with the geopm-service package

**Actual behavior**
They are installed with the geopm runtime package.

### PR DESCRIPTION
**Pull request by avilcheslopez**
- Fixes #2872

### CODE REVIEW

File: service/Makefile.am [hunk 1]
```diff
@@ -437,6 +437,7 @@ include test/Makefile.mk
 if !ENABLE_SANITIZERS
 include geopmdpy_test/Makefile.mk
 include integration/Makefile.mk
+include shell_completion/Makefile.mk
```
Comment (dannosliwcd) [inline service/Makefile.am]: Why do shell completion scripts depend on the sanitizer config?

File: service/Makefile.am [hunk 2]
```diff
@@ -437,6 +437,7 @@ include test/Makefile.mk
...
 endif
 
 include fuzz_test/Makefile.mk
```
Comment (cmcantalupo) [inline service/Makefile.am]:
```suggestion
endif

include shell_completion/Makefile.mk

include fuzz_test/Makefile.mk
```

**Review by dannosliwcd — DISMISSED** [review]

**Review by dannosliwcd — DISMISSED** [review]

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -137,15 +137,15 @@ jobs:
            python3 -m pip install -r service/requirements.txt
     - name: configure service dir
       working-directory: service
-      run: ./autogen.sh && ./configure
+      run: ./autogen.sh && ./configure || cat config.log && false
```
Comment (cmcantalupo) [inline .github/workflows/build.yml]:
```suggestion
      run: ./autogen.sh && ./configure || (cat config.log && false)
```

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -77,13 +77,13 @@ jobs:
       run: .github/include_guards.sh
     - name: configure service dir
       working-directory: service
-      run: ./autogen.sh && ./configure --${{ matrix.debug-flag }} --${{ matrix.service-asan-flag }}
+      run: ./autogen.sh && ./configure --${{ matrix.debug-flag }} --${{ matrix.service-asan-flag }}  || (cat config.log && false)
```
Comment (cmcantalupo) [inline .github/workflows/build.yml]:
```suggestion
      run: ./autogen.sh && ./configure --${{ matrix.debug-flag }} --${{ matrix.service-asan-flag }} || (cat config.log && false)
```

**Review by cmcantalupo — APPROVED** [review]

**Pull request merged**